rkt/images.go

```diff
@@ -378,7 +382,12 @@ func (f *fetcher) fetch(aciURL, ascURL string, ascFile *os.File) (*openpgp.Entit
 				return nil, nil, fmt.Errorf("error setting up temporary file: %v", err)
 			}
 			if err = f.downloadSignatureFile(ascURL, ascFile); err != nil {
-				return nil, nil, fmt.Errorf("error downloading the signature file: %v", err)
+				if err == errRetrySignature {
+					retrySignature = true
+					stdout("server requested deferring the signature download")
```

Comment:
yeah, when are you supposed to use `stdout` vs `fmt.Println`? I see both sprinkled everywhere.